ui_testing/elements.py

```diff
@@ -549,10 +551,22 @@
                                  'value': 'materialTypefield'},
         'article_filter': {'method': 'id',
                            'value': 'articlefield'},
-        'chnaged_on_filter': {'method': 'id',
-                              'value': 'start_createdAt'},
+        'test_plans_filter': {'method': 'id',
+                              'value': 'testPlanfield'},
+        'test_units_filter': {'method': 'id',
+                              'value': 'testUnitfield'},
+        'status_filter': {'method': 'id',
+                          'value': 'status_idfield'},
+        'departments_filter': {'method': 'id', 'value': 'departmentsfield'},
+        'created_on_filter': {'method': 'xpath',
+                              'value': '//div[@class="col-lg-4"]//div//input[@id="start_createdAt"]'},
+        'created_on_filter_end': {'method': 'xpath',
+                                  'value': '//div[@class="col-lg-4"]//div//input[@id="end_createdAt"]'},
         'test_date_filter': {'method': 'id',
                              'value': 'start_testDate'},
+        'test_date_filter_end': {'method': 'id', 'value': 'end_testDate'},
+        'shipment_date_filter_end': {'method': 'xpath',
```

Comment:
there is an id in the xpath

Reply:
yes, I used xpath as there's 2 elements with same id , test_Date in search bar 
and test_date in filter menu

Reply:
Now, I replaced it by id only and it worked, it fills both feilds in two cases (xpath or id)